src/pages/post-outwards/index.js

```diff
@@ -0,0 +1,251 @@
+import { useState, useEffect, useContext } from 'react'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { RemittanceOutwardsRepository } from 'src/repositories/RemittanceOutwardsRepository'
+import Table from 'src/components/Shared/Table'
+import { useResourceQuery } from 'src/hooks/resource'
+import { Grid } from '@mui/material'
+import * as yup from 'yup'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { useForm } from 'src/hooks/form'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import { RemittanceSettingsRepository } from 'src/repositories/RemittanceRepository'
+import { DataSets } from 'src/resources/DataSets'
+import FormShell from 'src/components/Shared/FormShell'
+import { ControlContext } from 'src/providers/ControlContext'
+import toast from 'react-hot-toast'
+
+const Postoutwards = () => {
+  const [data, setData] = useState([])
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const {
+    labels: _labels,
+    access,
+    refetch
+  } = useResourceQuery({
+    endpointId: RemittanceOutwardsRepository.Postoutwards.qry,
+    datasetId: ResourceIds.PostOutwards
+  })
+
+  const { formik } = useForm({
+    initialValues: {
+      countryId: '',
+      currencyId: '',
+      corId: '',
+      dispersalType: ''
+    },
+    access,
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      countryId: yup.string().required()
+    }),
+    onSubmit: async values => {
+      const checkedUserIds = data.filter(row => row.checked).map(row => row.recordId)
+
+      if (checkedUserIds.length > 0) {
+        try {
+          await postRequest({
+            extension: RemittanceOutwardsRepository.Postoutwards.post2,
+            record: JSON.stringify({ ids: checkedUserIds })
+          })
+        } catch (error) {}
+      }
+      fetchRemittanceData(
```

Comment:
u are passing params to fetchRemittanceData , but this function has no arguments